coderd/database/migrations/000328_app_group.down.sql

```diff
@@ -0,0 +1 @@
+alter table workspace_apps drop column display_group;
```

Comment:
called `display_group` because group is a keyword

this matches the `display_order` column, which is usually just called "order" but is _also_ a sql keyword